mne/beamformer/_lcmv.py

```diff
@@ -152,15 +152,18 @@ def make_lcmv(info, forward, data_cov, reg=0.05, noise_cov=None, label=None,
         If 'normal', rather than pooling the orientations by taking the norm,
         only the radial component is kept. If 'max-power', the source
         orientation that maximizes output source power is chosen.
+        If None, a vector beamformer is computed, combining the output for all
+        source orientations.
```

Comment:
@britta-wstnr thanks for looking into this.

Are you sure this is correct? Take a look here: https://github.com/mne-tools/mne-python/blob/master/mne/beamformer/_lcmv.py#L396. This also seems to depend on what is in the forward.

Reply:
You are right (maybe why it was left out before?). I will see how to rephrase it with taking the forward into account, thanks again!